djedi/plugins/img.py

```diff
@@ -1,16 +1,45 @@
 import json
 import six
+from PIL import Image
 from django.utils.html import escape
-from cio.plugins.base import BasePlugin
 from django.core.files.uploadedfile import InMemoryUploadedFile
+from django import forms
 from hashlib import sha1
 from os import path
 
+from .form import FormsBasePlugin, BaseEditorForm
 
-class ImagePluginBase(BasePlugin):
 
+class DataForm(BaseEditorForm):
+    attr_id = forms.CharField(
```

Comment:
This change current name `id` to `attr_id`, right? Doesn't this break existing persisted image nodes having content in storage named `id` to be "lost"? Gut-feeling says we must preserve current names, id, alt and class.

Reply:
Ok. I've reverted the names of the keys stored for additional data.